### ISSUE TITLE
svg attributes created with wrong namespace

### ISSUE THREAD

**Issue by vicb** [angular/angular#6255]
```html
<svg>
  <image x="20" y="20" width="300" height="80"  xlink:href="image.png" />
</svg>
```

Using beta.0 the above example results in the attributes x, y, width, height and xlink:href being created with the namespace http://www.w3.org/1999/xlink.

see https://github.com/angular/angular/pull/5989#issuecomment-168085340 for details

/cc @gionkunz

**Comment by vicb** [comment]
fixed via #5993

**Comment by gionkunz** [comment]
Awesome! :-)
